submissions/2025-09-02-tmQM-xtb-Dataset-T=100K-low-mw-high-coordinate-mult=3-v0.0/README.md

```diff
@@ -0,0 +1,61 @@
+# tmQM xtb Dataset T=100K low-mw high-coordinate mult=3 v0.0
+
+### Description
+
+This dataset was generated starting from the tmQM dataset (release 13Aug2024, https://github.com/uiocompcat/tmQM). 
+This dataset contains 6,425 unique systems with 206,240 total configurations / spin states below 600 Da.  The molecules are 
+limited to containing transition metals Pd, Zn, Fe, or Cu, and also only contain elements Br, C, H, P, S, O, N, F, Cl
+with charges: {-1,0,+1}. The metal is restricted to greater than three coordination sites for Pd, four for Fe, 
+and one for Cu and Zn. Each molecule was preprocessed using gfn2-xtb, and then a short MD simulation
+performed to provide ~30 additional configurations of per molecules at a multiplicity of 3. This
+singlepoint dataset was then run with the BP86/def2-TZVP for with those geometries from molecular dynamics using
+ gfn-xtb. Each configuration is reported with the following properties: 'energy', 'gradient', 'dipole', 'quadrupole',
+'wiberg_lowdin_indices', 'mayer_indices', 'lowdin_charges' 'dipole_polarizabilities', 'mulliken_charges'. SMILES
+strings where generated from tmos (https://github.com/openforcefield/tmos) when possible. These SMILES strings can be
+imported into RDKit for initial visualization, but will not reflect the coordinate geometries presented from tmQm.
+
+### General Information
+
+- Date: 2025-09-02
+- Purpose: BP86/def2-TZVP Conformers for single metal complexes with Pd, Fe, Zn, Cu, and change of {-1,0,+1} and multiplicity of 3. MW <= 600 Da, generally high coordinate, and 30 geometry samples
+- Dataset Type: singlepoint
+- Name: tmQM xtb Dataset T=100K low-mw high-coordinate mult=3 v0.0
+- Number of unique molecules: 6,425
+- Number of filtered molecules: 0
+- Number of Conformers: 206,240
+- Number of conformers (min mean max): 3, 30, 31
```

Comment:
I saw this in the last one too, I'm assuming the extra conformer is some kind of molecule overlap/clash? If so, it doesn't seem worth the effort to find out why some of them have 31 conformers.

Reply:
It's that there are 30 conformers that are off optimum in addition to the optimized geometry. Chris then filtered out conformers with bond distances >0.15 Å. It seems that on average one configuration was thrown out.